.coveragerc

```
@@ -435,6 +435,12 @@ omit =
     homeassistant/components/harmony/data.py
     homeassistant/components/harmony/remote.py
     homeassistant/components/harmony/util.py
+    homeassistant/components/hassmpris/__init__.py
+    homeassistant/components/hassmpris/cert_data.py
+    homeassistant/components/hassmpris/const.py
```

Why exclude these two from coverage?
They seem to be imported by config-flow so they are not uncovered.

```suggestion
    homeassistant/components/hassmpris/cert_data.py
    homeassistant/components/hassmpris/const.py
```

---

I will perform the changes you requested right away — with one request on my part:

I would ask you to reconsider the `intflag_to_string` — it was requested by another reviewer be put into `utils/enum` in this PR.  It is useful, and it simplifies reading logs when it comes to debugging state, as it avoids having to manually decode flags using a bit calculator.

In light of the above, what do you say?

---

It's all done.